docs/system-admin-guide/authentication/saml/README.md

```diff
@@ -237,20 +239,20 @@ SAML responses by identity providers are required to be signed. You can configur
 
 Use the key `attribute_statements` to provide mappings for attributes returned by the SAML identity provider's response to OpenProject internal attributes. 
 
-**a) Attribute mapping example for settings.yml**
+**a) Attribute mapping example for configration.yml**
 
 ```yaml
-# <-- other configuration -->
-# Attribute map in SAML
-attribute_statements:
-  # Use the `mail` attribute for 
-  email: ['mail']
-  # Use the mail address as login
-  login: ['mail']
-  # What attribute in SAML maps to the first name (default: givenName)
-  first_name: ['givenName']
-  # What attribute in SAML maps to the last name (default: sn)
-  last_name: ['sn']
+      # <-- other configuration -->
```

Comment:
I'd also repeat the parent nodes here for clarification
```suggestion
default:
  # <-- other configuration -->
    mysaml1:
      # <-- other configuration -->
```

Reply:
hi @oliverguenther i added these, thank you for reminding me that it is still not too accurate i was afraid its getting too long :) 

another question

in lines 335 and 348 and 349 there is a KOMMA ',' as the last CHARACTER,
in my understanding to YAML, i never seen that, is that correct?

Thank you

Reply:
Hi Oliver, I removed the 3 commas

now i see in line 358 there is a comma that might be a mistake too
or there shall be NO commas at all in that security {} abstract
waiting for your input